### PR TITLE
feat: version bump celestia-da to v0.12.9 and op stack version compat with v0.12.9

### LINKED ISSUE #1368: feat: upgrade optimism docs to v1.1.0-OP_op-node/v1.4.0-CN_v0.12.3
https://github.com/celestiaorg/optimism/releases/tag/v1.1.0-OP_op-node/v1.4.0-CN_v0.12.3

### PR DESCRIPTION
**Pull request by jcstein**
## Overview

Resolves #1368 

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

**Review by tuxcanfly — APPROVED** [review]

File: developers/optimism-devnet.md [hunk 1]
```diff
@@ -250,14 +250,14 @@ errors submitting data to Celestia.
...
-v1.0.0 release.
+[celestiaorg/optimism](https://github.com/celestiaorg/optimism/tree/release-v1.1.0)
+v1.1.0 release.
 
 The `op-batcher/batcher/driver.go` and
 `op-node/rollup/derive/calldata_source.go` files are part of the Ethereum
 fallback mechanism in the `op-batcher` and `op-node` respectively.
 
-In [`driver.go`, the `sendTransaction` function is responsible for the write path](https://github.com/celestiaorg/optimism/blob/release-v1.0.0/op-batcher/batcher/driver.go#L400-L406)
+In [`driver.go`, the `sendTransaction` function is responsible for the write path](https://github.com/celestiaorg/optimism/blob/9c6a1db78bc1da08b1e36548ebedf3d9a97e6dcc/op-batcher/batcher/driver.go#L359-L389)
```
Comment (tuxcanfly) [inline developers/optimism-devnet.md:260]:
```suggestion
In [`driver.go`, the `calldataTxCandidate` function is responsible for the write path](https://github.com/celestiaorg/optimism/blob/release-v1.1.0/op-batcher/batcher/driver.go#L405-L419)
```

Comment (tuxcanfly) [inline developers/optimism-devnet.md:260]:
```suggestion
In [`driver.go`, the `calldataTxCandidate` function is responsible for the write path](https://github.com/celestiaorg/optimism/blob/release-v1.1.0/op-batcher/batcher/driver.go#L405-L419)
```

File: developers/optimism-devnet.md [hunk 2]
```diff
@@ -292,7 +292,7 @@ func (l *BatchSubmitter) sendTransaction(
 ```
 
 In `calldata_source.go`,
-[the `DataFromEVMTransactions` function defines the read path](https://github.com/celestiaorg/optimism/blob/release-v1.0.0/op-node/rollup/derive/calldata_source.go#L138-L163)
+[the `DataFromEVMTransactions` function defines the read path](https://github.com/celestiaorg/optimism/blob/9c6a1db78bc1da08b1e36548ebedf3d9a97e6dcc/op-node/rollup/derive/calldata_source.go#L138-L185)
```
Comment (tuxcanfly) [inline developers/optimism-devnet.md:295]:
```suggestion
[the `DataFromEVMTransactions` function defines the read path](https://github.com/celestiaorg/optimism/blob/release-v1.1.0/op-node/rollup/derive/calldata_source.go#L102-L139)
```

**Review by tuxcanfly — APPROVED** [review]

**Comment by jcstein** [comment]
merging until new version live 🫡

**Pull request merged**